.github/workflows/handler-destroy.yml

```diff
@@ -0,0 +1,193 @@
+name: Pull Request Destroy Handler
+
+on:
+  repository_dispatch:
+    types:
+      - destroy-command
+
+jobs:
+  public_active_active:
+    name: Destroy resources from Public Active/Active test
+    if: ${{ contains(github.event.client_payload.slash_command.args.unnamed.all, 'all') || contains(github.event.client_payload.slash_command.args.unnamed.all, 'public-active-active') }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    env:
+      WORK_DIR_PATH: ./tests/public-active-active
+      AWS_DEFAULT_REGION: us-east-2
+    steps:
+      - name: Create URL to the run output
+        id: vars
+        run: echo ::set-output name=run-url::https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID
+
+      # Checkout the branch of the pull request being tested
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.event.client_payload.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.client_payload.pull_request.head.sha }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          persist-credentials: false
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          cli_config_credentials_hostname: 'app.terraform.io'
+          cli_config_credentials_token: ${{ secrets.PUBLIC_ACTIVE_ACTIVE_TFC_TOKEN }}
+          terraform_version: 0.14.8
+          terraform_wrapper: true
+
+      - name: Terraform Plan Destroy
+        id: plan-destroy
+        working-directory: ${{ env.WORK_DIR_PATH }}
+        run: terraform plan -destroy -out=destroy.tfplan
```

Comment:
I think that the use of the plan file may be unnecessary since we are using the remote backend: https://www.terraform.io/docs/cloud/run/modes-and-options.html#destroy-mode. More broadly, what do you think about omitting the separate plan stage and aligning with the `/test` workflow where we go directly to the apply stage without an explicit plan? It's slightly less complex and the plan output will still be included at the beginning of the destroy output.